src/backend/api/model/team.py

```diff
@@ -1,7 +1,7 @@
 # coding: utf-8
 import pymysql
 from util.setting import Setting
-import api.common.constant as constant
+from sqlalchemy.orm import relationship
 import api.common.sql as format
 import datetime
 import sys
```

Comment:
The 'to_dict' methods are removed. Ensure that this is intentional and does not break any functionality relying on these methods.